packages/webapp/src/containers/Crop/AddManagementPlan/RowMethod/index.js

```diff
@@ -10,10 +10,12 @@ export default function RowMethod({ history, match }) {
   const variety_id = match.params.variety_id;
   const variety = useSelector(cropVarietySelector(variety_id));
 
-  const persistPath = [`/crop/${variety_id}/add_management_plan/planting_method`];
+  const goBackPath = history.location.state?.from === undefined ?  `/crop/${variety_id}/add_management_plan/planting_method` : history.location.state.from;
```

Comment:
In what situation would `location.state.from` be `undefined`?

Reply:
It would be undefined if user comes to this page through pages other than LF-1587(InGroundTransplant)